sipXtackLib/src/net/SipSubscribeServer.cpp

```diff
@@ -1057,9 +1057,19 @@ UtlBoolean SipSubscribeServer::handleSubscribe(const SipMessage& subscribeReques
                        if (availableMediaTypes.isNull())
                        {
                           // No MIME types are available, so the resource does not exist.
-                          subscribeResponse.setResponseFirstHeaderLine(SIP_PROTOCOL_VERSION,
-                                                                       SIP_NOT_FOUND_CODE,
-                                                                       SIP_NOT_FOUND_TEXT);
+                          // If requested Id is Resource List, answer 200 OK instead of 404 NOT FOUND to prevent UA from Spamming
```

Comment:
A common lib is not the best part for specific RLS implementations, agreed. 

My problem was to find a point in the code to handle this inside the RLS, without breaking everything. For an empty Resource List the final point is when it the Subscribe Server detects the missing content. So if i want to keep the logic inside the RLS, i have to add a new return value and a handling for this specific value inside the subscribe server.

In the end it is an alternative special implementation, a bit more complex but even more common. 

Which way would you prefer?